tests/migration_test.go

```diff
@@ -116,11 +116,11 @@ import (
 )
 
 const (
-	fedoraVMSize         = "256M"
-	secretDiskSerial     = "D23YZ9W6WA5DJ487"
-	stressDefaultVMSize  = "100"
-	stressLargeVMSize    = "400M"
-	stressDefaultTimeout = 1600
+	fedoraVMSize               = "256M"
+	secretDiskSerial           = "D23YZ9W6WA5DJ487"
+	stressDefaultVMSize        = "100"
+	stressLargeVMSize          = "400M"
+	stressDefaultSleepDuration = 30
```

Comment:
```
// give stress tool some time to trash more memory pages before returning control to next steps
if stressTimeoutSeconds < 15 {
	time.Sleep(time.Duration(stressTimeoutSeconds) * time.Second)
} else {
	time.Sleep(15 * time.Second)
}
```
Anything above 15 seconds was already capped.
I am not convinced we should tweak this. Doesn't it mean we only run the stress for 30 seconds max? That in some cases might not be enough. WDYT?

Reply:
Yeah, I read this in a wrong way. You're absolutely right, thanks for catching that!